docs/source/setup/ssh.rst

```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
 The ``dask-ssh`` utility depends on the ``paramiko``::
 
     pip install paramiko
+
+
+CLI Options
+-------- 
```

Comment:
Sphinx will give a small warning unless the number of dashes matches the number of characters in the title like the following:

```
CLI Options
-----------
```